doc/content_item_fields.md

```diff
@@ -16,7 +16,8 @@ from the content store API can be found in
 
 ## `base_path`
 
-A string. Present in all contexts.
+A string. Present in the request URL in the publishing context, stored as `_id`
```

Comment:
:+1: I agree that the `_id` bit is an internal implementation detail that should never be surfaced to consumers of this API.  (The field is additionally [aliased as `base_path`](https://github.com/alphagov/content-store/blob/master/app/models/content_item.rb#L25) on the model, so even within the app, nothing should be referencing `_id` outside the model itself.)